feature_layers/service-feature-table-manual-cache/src/main/java/com/esri/samples/service_feature_table_manual_cache/ServiceFeatureTableManualCacheSample.java

```diff
@@ -117,8 +122,8 @@ public void start(Stage stage) {
         }
       });
 
-      // create a ArcGISMap with topographic basemap
-      ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+      // create a map with a basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
 
       // add feature layer to the ArcGISMap
       map.getOperationalLayers().add(featureLayer);
```

Comment:
feature_layers/feature-layer-query/src/main/java/com/esri/samples/feature_layer_query/FeatureLayerQuerySample.java does this in a slightly different order:
```
      // create a map with a basemap style
      final ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);

      // create a map view and set its map
      mapView = new MapView();
      mapView.setMap(map);

      // add feature layer to operational layers
      map.getOperationalLayers().add(featureLayer);
```

I quite like that one better to what's in this file, might be worth deciding between the two and sticking to one pattern?

Reply:
Agreed, amending